src/aerialmap_display.cpp

```diff
@@ -468,51 +466,81 @@ void AerialMapDisplay::assembleScene()
   tileCache_.purge({ tileId, blocks_ });
 }
 
+// rotation matrix for rotating the current frame to ENU
+Ogre::Matrix3 rotateToENU(int convention)
```

Comment:
nice! 

since everything in here is static, can the whole be static/constexpr?